src/schema/electronic-travel-authorisation.json

```diff
@@ -670,6 +678,14 @@
                     "type": "string",
                     "description": "Document number.",
                     "example": "75 127001"
+                },
+                "passportBioDataPagePhoto": {
+                    "typeRef": "#/domains/Generic/types/StructuredTextItems",
```

Comment:
StructuredTextItems = array, plural. Shouldn't this be a single item, most likely File?